docs/versioned_docs/version-2.11.0/troubleshooting/cstor.md

```diff
@@ -268,7 +268,7 @@ In such cases where pool import happens without cache file and if any of the dev
 **Troubleshooting**
 This might be encountered because of one of the following situations:
 
-1. The device that has gone bad is actually a part of the cStor pool on the node. In such cases, Block device replacement needs to be done, the detailed steps to it can be found [here](/docs/next/ugcstor-csi.html#a-class-anchor-aria-hidden-true-id-performance-tunings-in-cstor-pools-a-performance-tunings-in-cstor-pools).
+1. The device that has gone bad is actually a part of the cStor pool on the node. In such cases, Block device replacement needs to be done, the detailed steps to it can be found [here](/user-guides/cstor-csi#tuning-cstor-pools).
```

Comment:
```suggestion
1. The device that has gone bad is actually a part of the cStor pool on the node. In such cases, Block device replacement needs to be done, the detailed steps to it can be found [here](/deprecated/spc-based-cstor#setting-performance-tunings).
```